src/App/Fossa/Analyze.hs

```diff
@@ -327,9 +348,12 @@ uploadSuccessfulAnalysis (BaseDir basedir) apiOpts metadata jsonOutput override
   -- Warn on contributor errors, never fail
   void . Diag.recover . runExecIO $ tryUploadContributors basedir apiOpts (uploadLocator uploadResult)
 
-  if fromFlag JsonOutput jsonOutput
-    then logStdout . decodeUtf8 . Aeson.encode $ buildProjectSummary revision (uploadLocator uploadResult) buildUrl
-    else pure ()
+  _ <-
```

Comment:
does this compile without this line?

Reply:
So it does! Assumptions haunt me yet again.